packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -8203,6 +8257,29 @@ Specifies the Single Logout (SLO) behavior for a Custom SAML application
 }
 ```
 
+### Participate Single Logout object
+
+Specifies the Participate Single Logout behavior for a Custom SAML application
+
+| Property  | Description                                                                  | Datatype | Nullable |
+| --------- | ---------------------------------------------------------------------------- | -------- | -------- |
+| enabled             | Indicates whether the application supports `participateSlo`                    | Boolean  | FALSE    |
+| logoutRequestUrl    | URL where Okta sends the logout request                            | URL      | TRUE     |
+| bindingType         | HTTP binding used to send an `<LogoutRequest>` message to the RP   | `POST` or `REDIRECT`   | TRUE     |
+| sessionIndexRequired | Allows you to end a specific user session instead of all active user sessions | Boolean   | TRUE     |
```

Comment:
Same for SAML, would it be better to be more explicit and just say this boolean determines whether we send `SessionIndex` elements in the request?

Reply:
modified description to "Determines whether Okta sends SessionIndex in the logout request"